plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureStorageSettings.java

```diff
@@ -41,41 +41,41 @@
 import java.util.Set;
 
 public final class AzureStorageSettings {
-    // prefix for azure client settings
-    private static final String PREFIX = "azure.client.";
```

Comment:
Why remove the internal constant? Seems like this is just asking for a typo in the future in one setting to cause a storm of problems :/

Reply:
fair enough. I can add it back.